<ui version="4.0" >
 <class>SciDAVisAbout</class>
 <widget class="QDialog" name="SciDAVisAbout" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../appicons.qrc" >
    <normaloff>:/appicon</normaloff>:/appicon</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="version_label" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>0</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>SciDAVis XX.XX.XX-betaXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="release_date_label" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>40</y>
     <width>261</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Released XXXX-XX-XX</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="credits_box" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>239</red>
         <green>239</green>
         <blue>239</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>239</red>
         <green>239</green>
         <blue>239</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Close</string>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset resource="../appicons.qrc" >
     <normaloff>:/applogo</normaloff>:/applogo</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>130</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../appicons.qrc" />
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>SciDAVisAbout</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>251</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
